doc/contributing/maintaining-openssl.md

```diff
@@ -160,6 +165,16 @@ files if they are changed before committing:
 % git commit
 ```
 
+### OpenSSL 3.0.x
+
+```console
+    $ make -C deps/openssl/config
+    $ git add deps/openssl/config/archs
+    $ git add deps/openssl/openssl
```

Comment:
@richard this matches what is in the commit comments below and looks like we have done. It is probably also related to  https://github.com/nodejs/node/issues/42081 as git add deps/openssl/openssl adds all of the headers under `openssl/openssl/include` versus the selective ones that were added for OpenSSL 1.1.1. 

I don't think we want to change right now as we get releases out, but worth looking to see if possibly we can revert to adding just those three, find out why more were needed and possibly ask @danbev when he gets back.

Reply:
Actually I take that back since I guess the the examples were only showing the `git adds` that were necessary for a particular commit as opposed to what needs to be done in general. Therefore, it likely is not related to https://github.com/nodejs/node/issues/42081 after all.

Reply:
FWIW AFAICT the big difference between OpenSSL 1.1.1 and 3.0.x in Node.js is the `deps/openssl/config/archs` -- there's many more files in OpenSSL 3.0.x under there (see description in https://github.com/nodejs/node/issues/42081)  and a preliminary glance suggested a lot of duplication between the archs.